assignment-2/I20-2466-FATIMA-ANWAR.md

```diff
@@ -0,0 +1,8 @@
+Q1:
+Docker has a virtualised OS whereas VMs have a virtualised hardware.
+Docker has burstable memory while VMs have static memory.
+VMs have preinstalled libraries meanwhile in containers we need to install the libraries.
```

Comment:
wrong, containers have libraries installed in Images, in VMs we need to install libraries on OS